src/app.html

```diff
@@ -15,25 +15,32 @@
         <div class="headerConnectButton">
           <connectbutton></connectbutton>
         </div>
-        <div class="mobilemenuButton"><img click.delegate="toggleMobileMenu()" src="/Icon_Menu.svg" /></div>
-      </div>
-      <div class="mobileMenuContainer ${showingMobileMenu ? 'showingMobileMenu' : ''}">
-        <div class="menu">
-          <networkfeedback></networkfeedback>
-          <div class="mobileConnectButton">
-            <connectbutton hide-balances></connectbutton>
-          </div>
-          <div class="menuitems">
-            <header-buttons vertical on-navigate.call="onNavigate()"></header-buttons>
-          </div>
+        <div class="mobilemenuButton ${showingMobileMenu ? 'showingMobileMenu' : ''}" click.delegate="toggleMobileMenu()">
+          <span></span>
+          <span></span>
+          <span></span>
+          <span></span>
         </div>
       </div>
-      <div show.to-view="!showingMobileMenu" class="mainContent">
+      <div class="mainContent">
         <router-view></router-view>
       </div>
-      <div show.to-view="!showingMobileMenu" class="appFooter">
+      <div class="appFooter">
         <footer></footer>
       </div>
+      <div if.bind="showingMobileMenu" class="animateMobileMenuBackground au-animate">
```

Comment:
I had to move this `mobileMenuContainer` out from the grid, to the parent element in order to animate it on top of the page.